netlify/functions/deploy-succeeded.js

```diff
@@ -0,0 +1,43 @@
+const fetch = require('node-fetch');
+
+/**
+ * @param {object} event
+ * @param {string} event.body - https://jsoneditoronline.org/#left=cloud.fb1a4fa30a4f475fa6887071c682e2c1
+ */
+exports.handler = async (event) => {
+  const { payload } = JSON.parse(event.body);
+  const repo = payload.review_url.match(/github\.com\/(.*)\/pull\/(.*)/);
+  if (!repo) {
+    throw new Error(`No repo found at review_url: ${payload.review_url}`);
+  }
+
+  // eslint-disable-next-line no-console
+  console.info(`repo:`, repo[1]);
+  // eslint-disable-next-line no-console
+  console.info(`PR:`, repo[2]);
+  // eslint-disable-next-line no-console
+  console.info(`url:`, payload.deploy_ssl_url);
+
+  // for more details > https://circleci.com/docs/2.0/api-developers-guide/#
+  await fetch(`https://circleci.com/api/v2/project/gh/${repo[1]}/pipeline`, {
```

Comment:
A side though, the alternative option would be to:

1. `yarn docs:build`
2. Use something to serve in the CirleCI build the .html and .js files in `/docs/.next`
3. Run the e2e test against 

Pros:

  - In the same CircleCI workflow

Potential advantage:

- If we have better CircleCI machines, then `yarn docs:build` would build faster, but if we upgrade Netlify to a performance build, or migrate to Vercel with better build time, then that could be the opposite 

Disadvantage

- We build the docs twice

Reply:
I think if we can do this, that'd be great:
1. build the docs
2. run the e2e-test
3. deploy to netlify
all of these steps are in CircleCI